x/staging/posts/keeper/common_functions.go

```diff
@@ -30,6 +30,23 @@ func (k Keeper) IteratePosts(ctx sdk.Context, fn func(index int64, post types.Po
 	}
 }
 
+func (k Keeper) IterateCommentIDsByPost(ctx sdk.Context, postID string, fn func(index int64, commentID string) (stop bool)) {
```

Comment:
I think this is only used inside `GetPostCommentsIDs`, which appears to be unused as well. Maybe we can delete this as well?